services/QuillLMS/client/app/bundles/Proofreader/components/proofreaderActivities/editTooltip.tsx

```diff
@@ -0,0 +1,127 @@
+import * as React from 'react'
+
+import {
+  UNNECESSARY_SPACE,
+  MULTIPLE_UNNECESSARY_DELETION,
+  SINGLE_UNNECESSARY_DELETION,
+  MULTIPLE_UNNECESSARY_ADDITION,
+  SINGLE_UNNECESSARY_ADDITION,
+  UNNECESSARY_CHANGE
+} from '../../helpers/determineUnnecessaryEditType'
+import useFocus from '../../../Shared/hooks/useFocus'
+
+const notNecessaryIconSrc = `${process.env.CDN_URL}/images/icons/review-not-necessary.svg`
+const incorrectIconSrc = `${process.env.CDN_URL}/images/icons/review-incorrect.svg`
+const correctIconSrc = `${process.env.CDN_URL}/images/icons/review-correct.svg`
+
+const renderConceptExplanation = (concept) => {
+  if (!(concept && concept.explanation)) { return <span /> }
+  return (
+    <div className="explanation">
+      <p className="label">Explanation</p>
+      <p dangerouslySetInnerHTML={{ __html: concept.explanation }} />
+    </div>
+  )
+}
+
+const renderCorrectAnswers = (displayText) => {
+  if (!displayText) { return }
+
+  const correctAnswerArray = displayText ? displayText.split('~') : []
+  const correctAnswers = correctAnswerArray.map(ca => <span className="correct-answer" key={ca}>{ca}</span>)
+  const correctAnswerHTML = <p>{correctAnswers}</p>
+  return (
+    <div aria-hidden={true}>
+      <p className="label">Correct</p>
+      {correctAnswerHTML}
+    </div>
+  )
+}
+
+const renderNotNecessaryExplanation = (state) => {
+  const unnecessaryArray = [UNNECESSARY_SPACE, MULTIPLE_UNNECESSARY_DELETION, SINGLE_UNNECESSARY_DELETION, MULTIPLE_UNNECESSARY_ADDITION, SINGLE_UNNECESSARY_ADDITION, UNNECESSARY_CHANGE]
+  if (!unnecessaryArray.includes(state)) { return }
+  let explanation
+
+  switch (state) {
+    case UNNECESSARY_SPACE:
+      explanation = 'You added an unnecessary space.'
+      break
+    case MULTIPLE_UNNECESSARY_DELETION:
+      explanation = "You took out words that weren't part of an error."
+      break
+    case SINGLE_UNNECESSARY_DELETION:
+      explanation = "You took out a word that wasn't part of an error."
+      break
+    case MULTIPLE_UNNECESSARY_ADDITION:
+      explanation = "You added unnecessary words."
+      break
+    case SINGLE_UNNECESSARY_ADDITION:
+      explanation = "You added an unnecessary word."
+      break
+    case SINGLE_UNNECESSARY_ADDITION:
+      explanation = "You added an unnecessary word."
+      break
+    case UNNECESSARY_CHANGE:
+      explanation = 'You made an unnecessary change.'
+      break
+  }
+
+  return <p className="unnecessary-explanation">{explanation}</p>
+}
+
+
+const EditTooltip = ({ state, id, tooltipHeight, back, numberOfEdits, next, index, displayText, concept, incorrectText, }) => {
+  const [containerRef, setContainerFocus] = useFocus()
+
+  React.useEffect(() => {
+    setContainerFocus()
+  }, [])
+
+  let src, headerText, altText
```

Comment:
maybe I'm missing something, but seems like we're not doing anything with this `altText` variable?